tuxemon/condition/condition.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, save_data: Optional[Mapping[str, Any]] = None) -> None:
 
         self.instance_id = uuid.uuid4()
         self.steps = 0.0
+        self.bond = False
         self.counter = 0
         self.cond_id = 0
         self.animation = Optional[str]
```

Comment:
I know it's not part of this PR, but since it looks like it was part of the previous one (split condition): isn't there a mistake here?
Didn't you want to do this instead:
```py
self.animation: Optional[str] = None
```
or
```py
self.animation: str = ""
```

Reply:
yes, there is the same to update in technique.py too